docker-compose-prod.yaml

```diff
@@ -0,0 +1,44 @@
+version: '3.8'
```

Comment:
delete this file. Integrate docker-compose profiles in existing docker file so that same file works in both cases.
Basically, profile `local` would use dev docker file `Dockerfile` and spin up postgres along with it.

And without profile specification, it should compile to production where it will use `infra/Dockerfile.prod` and _**not**_ spin up any database container.

Reply:
Plus prod file would rely on purely env, and not on `.env` file.